i18n/de.json

```diff
@@ -19,7 +19,8 @@
     "vi": "Vietnamesisch (Tiếng Việt)",
     "ru": "Russisch (Русский)",
     "ko": "Koreanisch (한국어)",
-    "pl": "Polnisch (Polski)"
+    "pl": "Polnisch (Polski)",
+    "nb-no": "Norwegisch (Norsk)"
```

Comment:
To be 100% correct, yes. IMO it wont have much benefits before someone adds nn-no (nynorsk).